packages/react-sdk/src/component-generator.ts

```diff
@@ -288,6 +288,9 @@ export const generateJsxChildren = ({
   let generatedChildren = "";
   for (const child of children) {
     if (child.type === "text") {
+      if (child.placeholder) {
```

Comment:
I would keep it rendered so user see consistent content in canvas and builder

Reply:
so would need to filter out placeholders before passing to this function?

Reply:
no need to filter at all, the flag is for ai only

Reply:
ah yes, you are right. need to filter before passing, forgot it is used in jsx utils